srcpkgs/scenefx/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'scenefx'
+pkgname=scenefx
+version=0.1
+revision=1
+archs="i686 x86_64"
```

Comment:
why?

Reply:
Whoops, looks like that slipped through the `xnew` template.  Corrected now.